mongo/integration/initial_dns_seedlist_discovery_test.go

```diff
@@ -63,6 +67,38 @@ func runSeedlistDiscoveryDirectory(mt *mtest.T, subdirectory string) {
 	}
 }
 
+// runSeedListDiscoverPingTest will create a new connection using the test URI and attempt to "ping" the server.
+func runSeedListDiscoveryPingTest(mt *mtest.T, tcase seedlistTest) {
+	mt.Parallel()
+
+	if !tcase.Ping {
+		return
+	}
```

Comment:
Optional: Consider moving this check to the calling function so that this function is only called if the test calls for running a ping op.